packages/contracts/docs/osx/01-how-it-works/02-framework/02-plugin-management/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Managing Plugins
+title: Plugins
```

Comment:
Will it be enough on the side menu? I guess, if the parent already makes it clear